tests/test_models_user.py

```diff
@@ -0,0 +1,113 @@
+from nose.tools import *
+
+from framework import auth
+from framework.auth import exceptions
+from website import models
+from tests import base
+from tests import factories
+
+
+class UserTestCase(base.OsfTestCase):
```

Comment:
I moved the file and have been moving tests that break there when I review them.